bats/katello-bats

```diff
@@ -61,6 +65,11 @@ then
     BATS_ARGS="$BATS_ARGS fb-virt-whom.bats"
 fi
 
+if [ "$swapfile" == true ];
+then
+    BATS_ARGS="$BATS_ARGS swapfile.bats"
```

Comment:
I think this would run at the end, after all the tests - maybe `BATS_ARGS="swapfile.bats $BATS_ARGS "` instead?